engines/rbd.c

```diff
@@ -529,6 +532,11 @@ static int fio_rbd_queue(struct thread_data *td, struct io_u *io_u)
 static int fio_rbd_init(struct thread_data *td)
 {
 	int r;
+	struct rbd_data *rbd;
+	rbd = (struct rbd_data*)(td->io_ops_data);
+	if(rbd && rbd->connected) {
```

Comment:
Can 'rbd' ever be NULL here?

Also style - please follow the general style in the file. In this csae, space after the 'if'.